ui/perfherder/graphs/TestDataModal.jsx

```diff
@@ -220,7 +219,7 @@ export default class TestDataModal extends React.Component {
       })
       .filter(
         (item) =>
```

Comment:
`item` is very generic & doesn't give any immediate clue on what it's holding. Could you give it a better name?

Reply:
`responseSeries` would be better?

Reply:
Yes, it is better.